candig/server/cli/server.py

```diff
@@ -23,9 +20,9 @@ def __init__(self, app, options=None):
 
     def load_config(self):
         config = dict(
-            [(key, value) for key, value in self.options.iteritems()
+            [(key, value) for key, value in self.options.items()
```

Comment:
config = {key: self.options[key] for key in self.options
            if key in self.cfg.settings and self.options[key] is not None}

Reply:
Iterating through the pair seems fine, which eliminates the need of a dict look up?

Reply:
Ya I think it's more pythonic to use dict() with the list of (k, v) tuples here

Reply:
okay